### DISCUSSION TITLE
Dynamically Run multiple queries

### DISCUSSION THREAD

**Discussion by Annie-khalid** [supabase/supabase#879] [Questions] [answered]
I want to run multiple queries dynamically,  my tables have foreign relations ... multiple query is working on single table but not on foreign table.  What is the best way to run/check conditions on foreign table attributes ?  Thanks

**Comment by steve-chavez** [comment, marked as answer]
You can query foreign relations with: https://supabase.io/docs/client/select#query-foreign-tables

And for more complex queries you can use stored procedures: https://supabase.io/docs/client/rpc
